imap_processing/codice/codice_l0.py

```diff
@@ -15,21 +15,22 @@
     packet_list = decom_packets(packet_file, xtce_document)
 """
 
-from imap_processing import decom, imap_module_directory
+from imap_processing import decom
 
 
-def decom_packets(packet_file: str) -> list:
+def decom_packets(packet_file: str, xtce_document) -> list:
```

Comment:
Rather than passing in the XTCE document everywhere you want to use this would it be easier to keep a mapping in this and then pass in a string of the type you want to decommutate? That way only this one file needs to know the path locations and not the tests too.

```suggestion
_PACKET_TYPES = {"housekeeping": path_to_hk_definition, ...}
def decom_packets(packet_file: str, packet_type="housekeeping") -> list:
    if packet_type not in _PACKET_TYPES:
        raise ...
    xtce_document = _PACKET_TYPES[packet_type]
```

Reply:
Thanks, great suggestion. I didn't feel great about adding this parameter in, I like this solution better!